doc/tunnel/SAI-Proposal-EVPN-Multihoming.md

```diff
@@ -0,0 +1,616 @@
+     */
+    SAI_NEXT_HOP_ATTR_TUNNEL_ID,
+...
+```
+
+New Bridge-port of type next-hop group is introduced for FDB entries to point to the group of remote VTEPs.
+
+```
+typedef enum _sai_bridge_port_type_t
+...
+    /** Bridge port next hop group */
+    SAI_BRIDGE_PORT_TYPE_BRIDGE_PORT_NEXT_HOP_GROUP,
+} sai_bridge_port_type_t;
+```
+
+The Bridge-port of type next-hop group should have next-hop group ID attribute set.
+```
+typedef enum _sai_bridge_port_attr_t
+{
+...
+    /**
+     * @brief Associated bridge port nexthop group id
+     *
+     * @type sai_object_id_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @objects SAI_OBJECT_TYPE_NEXT_HOP_GROUP
+     * @condition SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_BRIDGE_PORT_NEXT_HOP_GROUP
+     */
+    SAI_BRIDGE_PORT_ATTR_BRIDGE_PORT_NEXT_HOP_GROUP_ID,
+...
+}
+```
+
+### 3.2.2 Designated Forwarder support
+
+A new bridgeport attribute defined to drop BUM traffic egressing the bridgeport. The default value of this 
+will be false. 
+Note: Traffic dropped due to this should not be counted against SAI_PORT_STAT_IF_OUT_DISCARDS i.e Interface Tx drop counters.
+
+```
+typedef enum _sai_bridge_port_attr_t
+{
+...
+
+    /**
+     * @brief Indicates if the bridge port is set to drop the Tunnel Terminated broadcast, unknown unicast and multicast traffic
+     * When set to true, egress BUM traffic will be dropped
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     * @validonly SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_PORT or SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_SUB_PORT or SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_TUNNEL
+     */
+    SAI_BRIDGE_PORT_ATTR_TUNNEL_TERM_BUM_TX_DROP,
+```
+
+To handle DF election per VLAN and when a dot1q bridge model is used the above attribute is defined as part of the Vlan Member. 
+ 
+```
+typedef enum _sai_vlan_member_attr_t
+{
+...
+
+    /**
+     * @brief Indicates if the bridge port is set to drop the Tunnel terminated broadcast, unknown unicast and multicast traffic
+     * When set to true, egress BUM traffic will be dropped
+     *  
+     * Valid only when the SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID is of type PORT.
+     * 
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     * 
+     */
+    SAI_VLAN_MEMBER_ATTR_TUNNEL_TERM_BUM_TX_DROP,
+
+```
+
+### 3.2.3 Split Horizon support
+
+  - Tunnels of peer mode P2P are only considered here. The isolation group object is used to achieve the split horizon functionality.
+  - There is no change to the Isolation group and group member definition.
+  - Bridgeport of type TUNNEL will also now have the isolation group attribute set.
+  - The isolation group members will be the client side bridge ports which share the ESI with the peering VTEPs.
+
+### 3.2.4 Fast Failover support
+
+A new Bridge port attribute (for type PORT) is introduced to specify the protection nexthop group ID.

```

Comment:
Please be explicit and use 'SAI_BRIDGE_PORT_TYPE_PORT' when you say PORT. As all these cases are applicable to port and lag.

Reply:
Ack.